apps/costcontrol/js/settings/settings.js

```diff
@@ -119,8 +120,12 @@ var Settings = (function() {
         closeSettings();
       });
 
-      function _setResetTimeToDefault(value, old, key, settings) {
+      function _setParametersToDefault(value, old, key, settings) {
```

Comment:
Marina, `_setParametersToDefault` is still a little bit generic. Can we explain in the name of the method what are we doing?